### DISCUSSION TITLE
中央附近的麵包店

### DISCUSSION THREAD

**Discussion by THH1118** [1101-GS4719-C/Python-Community-GS4719-C#1679] [General]
前幾天突然很想吃麵包，隨手查了一下發現學校附近有一間很大間的麵包店，除了有麵包、飲料聽說也有賣早午餐，可能是因為方向跟市區相反，位置也蠻低調的所以一直都沒注意到，所以就決定去看看。
![IMG_4937](https://github.com/NCU-GS4719-Python/Python-Community/assets/144879896/372b40b4-d59e-4f8a-b487-6ee6df43f5f3)
結果裡面意外的非常多人，店外面也停了很多汽車。
![IMG_4938](https://github.com/NCU-GS4719-Python/Python-Community/assets/144879896/34b69c33-d9ff-476c-a950-15803b8835b9)
我點的是咖哩雞丹麥，但我是外帶回家自己加熱的，因為內用有低消200覺得太貴了哈哈，超級好吃的。
這間店名字叫厚冶，有興趣的可以去搜尋看看。

**Comment by chia-kai-chang** [comment]
這家我吃過，停車空間很大，內用空間滿舒服的~

麵包部分，同行友人是說佛卡夏很道地，但是我木嘴吃不吃來~

然後我也覺得價位有點高~ 😂

**Comment by vu45tp6** [comment]
捲烘焙，離學校不遠，雖然店面不大，但都是手工烘焙的，而且品項從歐式到台式基本上都有，而且還有集點卡制度，會讓人有莫名的成就感。

**Comment by brianou583** [comment]
欸我超愛吃麵包，感謝推薦，雖然低消200確實不能常吃。但感覺是很厲害的店。
低調又在市區反方向的小店常常都藏著驚喜，根本美食寶藏點。
